inventories/netplan_cluster_example.yaml.j2

```diff
@@ -0,0 +1,23 @@
+# This netplan files defines the path of the cluster network which is managed
+# by systemd. Especially, it attributes its cluster IP addr to each node.
+network:
+  version: 2
```

Comment:
Defined the full Open vSwitch team0 bridge here.
Refer to: https://github.com/canonical/netplan/blob/main/examples/openvswitch.yaml